wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession.swift

```diff
@@ -451,6 +489,11 @@ public final class ZMUserSession: NSObject {
         restoreDebugCommandsState()
         configureRecurringActions()
 
+        if !ProcessInfo.processInfo.isRunningTests {
+            setupAnalyticsSession()
+            self.analyticsSession?.startSession()
+        }
```

Comment:
Can't it pass a boolean to the setup func? The test call the function themselves and just could pass `false`, no need to look into the process, or do I miss something 🤔

<img width="508" alt="Screenshot 2024-07-09 at 10 44 59" src="https://github.com/wireapp/wire-ios/assets/7196958/daae5dcd-2430-4a01-a644-2114a6d0410a">

Reply:
Thought about it but I find the ProcessInfo case more explicit.

Reply:
what about not passing the analyticsSessionConfiguration in tests to avoid this setup?

Reply:
It's just my suggestion, so feel free to choose yourself :)